### PR TITLE
unified/novacustom/wifi-bt-switch.md: added documentation

### PR DESCRIPTION
**Pull request by Mixss**

### CODE REVIEW

File: docs/unified/novacustom/wifi-bt-switch.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# WiFi + Bluetooth radios switch
+
+Novacustom laptops have the option to disable the WiFi + Bluetooth card
+completely. Afterdisabling this option, the power to the
```
Comment (macpijan) [inline docs/unified/novacustom/wifi-bt-switch.md:4]:
```suggestion
completely. After disabling this option, the power to the
```

File: mkdocs.yml [hunk 2]
```diff
@@ -94,6 +94,7 @@ nav:
           - 'Fn lock hotkey': unified/novacustom/fn-lock-hotkey.md
           - 'Battery management': unified/novacustom/battery-management.md
           - 'Fan profiles': unified/novacustom/fan-profiles.md
+          - 'WiFi + Bluetooth radios switch': unified/novacustom/wifi-bt-switch.md
```
Comment (miczyg1) [inline mkdocs.yml:97]: Adding a separate page for every option doesn't look like a good idea. Why not simply add a section/paragraph in the setup menu documentation? We already added many things that way here for novacustom and contribute to more scattered information each time

**Comment by miczyg1** [comment]
@Mixss @mkopec this PR needs updates/review addressing

File: mkdocs.yml (hunk 2 above)
Comment (mkopec) [reply mkdocs.yml:97]: Agreed, moved to shared setup menu documentation.

File: docs/unified/novacustom/wifi-bt-switch.md (hunk 1 above)
Comment (mkopec) [reply docs/unified/novacustom/wifi-bt-switch.md]: File removed, comment no longer applicable.

File: docs/dasharo-menu-docs/dasharo-system-features.md [hunk 3]
```diff
@@ -35,6 +35,10 @@ This menu offers security-sensitive options like:
   problems with DMA/IOMMU during OS boot, keep it disabled.
   NOTE: Windows may not boot if the option is enabled.
 
+- `Enable Wi-Fi + BT radios` - (applicable to devices with Wi-Fi and Bluetooth
+  cards) - Enables or disables radios for Wi-Fi and Bluetooth. When disabled,
+  the slot is unconfigured and power to the wireless module is disconnected.
```
Comment (miczyg1) [inline docs/dasharo-menu-docs/dasharo-system-features.md:40]:
```suggestion
- `Enable Wi-Fi + BT radios` - (applicable to laptops) - Enables or disables
  radios for Wi-Fi and Bluetooth. When disabled, the slot is unconfigured and
  power to the wireless module is disconnected.
```

Desktops typically have multiple PCIe slots, so the card may basically be in any slot and it is not always possible to cut off the power from it. Also for Protectli devices, it is not possible to cut off power, despite the WiFi slot has fixed location/Root Port.

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/dasharo-menu-docs/dasharo-system-features.md (hunk 3 above)
Comment (mkopec) [reply docs/dasharo-menu-docs/dasharo-system-features.md]: Suggestion applied in f888a04cad7509319e4a4cb0b8b7605ef959d268

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**